dsweb/src/test/java/com/doctusoft/dsw/client/gwt/TestTabSheetRenderer.java

```diff
@@ -37,7 +39,36 @@ public void testTabTitleChange() {
 		assertEquals("Tab2", JQuery.select("#tabsheet > ul.nav-tabs > li > a").text());
 		assertEquals(1, JQuery.select("#tabsheet > div.tab-content > div.tab-pane > #tabContent").length());
 	}
-
+	
+	@Test
+	public void testTabTitleComponentChange() {
+		Link tabTitle = new Link("ComponentTitle");
+		final BaseContainer tabCaption = new BaseContainer("li").add(tabTitle);
+		Tab tab = new Tab().withTitleComponent(tabCaption).withContent(new Container().withId("tabContent"));
+		TabSheet tabSheet = new TabSheet().withTab(tab).withId("tabsheet");
+		registerApp(tabSheet);
+		Link tabTitleSecond = new Link("Tab2");
```

Comment:
@epfisztner én azért leasszertálnám itt is, hogy a ComponentTitle megjelent-e, mert így most nem fut a teszt arra a Renderer ágra, ami az első hozzáadáskor meglévő titleComponent-et hozzáadja